src/nvim/tui/tui.c

```diff
@@ -350,6 +352,13 @@ static void tui_resize(UI *ui, int width, int height)
   data->scroll_region.left = 0;
   data->scroll_region.right = width - 1;
   data->row = data->col = 0;
+  const char *term = os_getenv("TERM");
+  if (term && (STARTS_WITH(term, "xterm") || STARTS_WITH(term, "rxvt"))) {
+      // try to resize the terminal window
+      char r[16]; // enough for 9999x9999
+      sprintf(r, "\x1b[8;%d;%dt", height, width);
```

Comment:
I'm not sure, actually, I just wanted to be safe... 

I just tried it with the linux terminal, and it simply ignored the control sequence, so I think removing the check is safe. 

I think libvterm doesn't set $TERM by itself, but I'm not sure.